sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -37,3 +37,8 @@ include::modules/sandboxed-containers-triggering-installation-kata-runtime.adoc[
 include::modules/sandboxed-containers-selecting-nodes.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-scheduling-workloads.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-viewing-workloads-from-cli.adoc[leveloffset=+2]
+
+== Additional resources
```

Comment:
I think this should be mentioned earlier. Maybe under "Preparing your cluster for OpenShift sandboxed container". Additional resources here to me would mean something that is not related but helpful.